driver/network/local/local.go

```diff
@@ -172,12 +179,129 @@ func (n *LocalNetwork) createNode(nodeConfig *node.OperaNodeConfig) (*node.Opera
 	return node, nil
 }
 
-// CreateNode creates non-validator nodes in the network.
+// CreateNode creates nodes in the network during run.
 func (n *LocalNetwork) CreateNode(config *driver.NodeConfig) (driver.Node, error) {
+	newValId := 0
+	if config.Validator {
+		log.Printf("Creating validator node %s", config.Name)
+		rpcClient, err := n.DialRandomRpc()
+		if err != nil {
+			return nil, err
+		}
+		defer rpcClient.Close()
+
+		// get a representation of the deployed contract
+		SFCContract, err := contract.NewSFC(common.HexToAddress("0xFC00FACE00000000000000000000000000000000"), rpcClient)
```

Comment:
Instance of putting SFC address here, it would be more readable and reusable to have it as a global variable.